### ISSUE TITLE
建议官方示例页面添加功能效果缩略图

### ISSUE THREAD

**Issue by Southjor** [openlayers/openlayers#14552]
比如这样的，查看会比较方便：

![image](https://user-images.githubusercontent.com/33611613/223897094-587f3562-859a-4979-96af-63f66eecadd7.png)

**Comment by tschaub** [comment]
I agree that would be nice.  We could use puppeteer to create screenshots when building the examples.  Probably wouldn’t want to do that in dev mode though.  We could also auto-generate them only if they don’t exist (allowing them to be manually generated).

**Comment by Southjor** [comment]
非常感谢！
